### PR TITLE
yay: add page

### PR DESCRIPTION
**Pull request by MasterTos**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/yay.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# yay
+
+> Yet Another Yogurt - A utility for Arch Linux to build and install packages from the Arch User Repository.
```
Comment (agnivade) [inline pages/linux/yay.md]: @waldyrious / @sbrl  - Is it correct to have a capital letter after `-` ?

File: pages/linux/yay.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+- Search and install from the repos and the AUR using yogurt mode:
```
Comment (agnivade) [inline pages/linux/yay.md]: Search and install _packaged_

**Comment by Foxboron** [comment]
@Morganamilo is better to ping for comments regarding yay

File: pages/linux/yay.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# yay
+
+> Yet Another Yogurt - A utility for Arch Linux to build and install packages from the Arch User Repository.
+
+- Search and install packaged from the repos and the AUR using yogurt mode:
```
Comment (Morganamilo) [inline pages/linux/yay.md]: >packaged

Typo

>yogurt mode

I guess the man page does call it this. I prefer "interactive search and install" myself and may even change it in the man page at some point.

I'd go for "Interactivity search and install packages from the repos and AUR". And if you really want to you could tack on (yogurt mode).

File: pages/linux/yay.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+`yay {{package_name}}`
+
+- Synchronize and update all packages (includes AUR):
+
+`yay -Syu`
+
+- Synchronize and update only AUR packages:
+
+`yay -Sua`
```
Comment (Morganamilo) [inline pages/linux/yay.md]: Good job on not falling into the trap of `yay -Syua`.

File: pages/linux/yay.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+- Install a new package (includes AUR):
+
+`yay -S {{package_name}}`
+
+- Remove a package and its dependencies (includes AUR packages):
+
+`yay -Rs {{package_name}}`
```
Comment (Morganamilo) [inline pages/linux/yay.md]: It is purely pacman's behaviour, I don't see a need to mention it here.

File: pages/linux/yay.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+- Search the package database for a keyword (includes AUR):
+
+`yay -Ss {{keyword}}`
+
+- List all currently installed packages (includes AUR packages):
+
+`yay -Qs`
```
Comment (Morganamilo) [inline pages/linux/yay.md]: It is purely pacman's behaviour, I don't see a need to mention it here.

**Comment by Morganamilo** [comment]
Also one thing you might want to do is list some of the more useful flags. I'm not sure how big/small tldr-pages are meant to be so I'll leave it to your discretion.

**Comment by sbrl** [comment]
We ok to merge, @Morganamilo?

**Comment by Morganamilo** [comment]
The -Rs and -Qs are still there. Yay does not do anything extra here so it literally just calls pacman. Additionally the (includes AUR packages) message does not really make any sense as these operations only act on the local database.

**Comment by agnivade** [comment]
@MasterTos - Could you make those changes ? Since you resolved those comments, I thought those were taken care of.

**Comment by agnivade** [comment]
@MasterTos - Could you make those changes ? Since you resolved those comments, I thought those were taken care of.

**Comment by agnivade** [comment]
@MasterTos - Could you make those changes ? Since you resolved those comments, I thought those were taken care of.

**Comment by agnivade** [comment]
@MasterTos - Could you make those changes ? Since you resolved those comments, I thought those were taken care of.

**Comment by agnivade** [comment]
@MasterTos - Could you update the PR with the requested changes ? I thought the changes were complete since the comments were resolved.

**Comment by agnivade** [comment]
@MasterTos - Could you update the PR with the requested changes ? I thought the changes were complete since the comments were resolved.

**Comment by agnivade** [comment]
@MasterTos - Could you update the PR with the requested changes ? I thought the changes were complete since the comments were resolved.

**Comment by agnivade** [comment]
@MasterTos - Could you update the PR with the requested changes ? I thought the changes were complete since the comments were resolved.

**Comment by agnivade** [comment]
@MasterTos - Could you update the PR with the requested changes ? I thought the changes were complete since the comments were resolved.

**Comment by agnivade** [comment]
Sorry, due to the github outage today. My comments appeared hundreds of times. I have deleted the duplicate ones.

**Comment by agnivade** [comment]
@MasterTos - I think @Morganamilo wanted to remove the -Rs and -Qs options. We can remove that and add a subheading - "Also see `pacman`".

**Comment by agnivade** [comment]
Awesome, thanks ! @Morganamilo - any further comments ?

**Comment by Morganamilo** [comment]
Whoops, a little late on the reply. Looks good, nice job.

**Pull request merged**